openedx/features/genplus_features/genplus/api/v1/views.py

```diff
@@ -189,6 +189,21 @@ def add_my_class(self, request, group_id=None):  # pylint: disable=unused-argume
             return Response(SuccessMessages.CLASS_REMOVED_FROM_FAVORITES.format(class_name=gen_class.name),
                             status=status.HTTP_204_NO_CONTENT)
 
+    @action(detail=True, methods=['put'])
```

Comment:
@zainvrk can we move retrieve method of ClassViewSet from genplus to genplus_learning and its genplus_learning model and serializer imports?

Reply:
Sure